R/get_statistic.R

```diff
@@ -76,6 +76,32 @@ get_statistic.default <- function(x, column_index = 3, verbose = TRUE, ...) {
   out
 }
 
+#' @export
+get_statistic.htest <- function(x, ...) {
+
+  if (x$method == "Fisher's Exact Test for Count Data") {
+    out <- data.frame(
+      Parameter = x$data.name,
+      Statistic = x$p.value,
```

Comment:
Is this right? Why don't we call this column `p`?

For what it's worth, we don't have a `Statistic` column for this test in `parameters()` output.

So maybe we can also return `NA_real_`?

``` r
library(parameters)

TeaTasting <- matrix(c(3, 1, 1, 3),
  nrow = 2,
  dimnames = list(
    Guess = c("Milk", "Tea"),
    Truth = c("Milk", "Tea")
  )
)

m <- fisher.test(TeaTasting, alternative = "greater")
parameters(m)
#> Fisher's Exact Test for Count Data
#> 
#> Odds.Ratio | CI_low |     p
#> ---------------------------
#> 6.41       |   0.31 | 0.243
#> 
#> Alternative hypothesis: true odds ratio is greater than 1
```

<sup>Created on 2022-01-24 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>

Reply:
My thinking is that for this test, the test statistic and p value are the same thing. It's a weird edge case, but seemed like the best option versus NA. We don't have a function for returning p values from model objects. Should we include p and perhaps parameter values for htest?

Reply:
@strengejacke What's your opinion on this? After this comment is resolved, we can merge this PR.